python/ray/experimental/state/api.py

```diff
@@ -597,7 +688,32 @@ def list_actors(
     detail: bool = False,
     raise_on_missing_output: bool = True,
     _explain: bool = False,
-):
+) -> List[Dict]:
+    """List actors in the cluster.
+
+    Args:
+        address: The IP address and port of the head node. Defaults to
+            `http://localhost:8265` if not set.
+        filters: List of tuples of filter key, predicate (=, or !=), and
+            the filter value. E.g., `("id", "=", "abcd")`
+        limit: Max number of entries returned by the state backend.
+        timeout: Max timeout value for the http/gRPC requests made.
+        detail: When True, more details info (specified in `ActorState`)
+            will be queried and returned. See
+            :ref:`ActorState <state-api-schema-actor>`.
+        raise_on_missing_output: When True, exceptions will be raised if
+            there is missing data due to truncation/data source unavailable.
+        _explain: Print the API information such as API latency or
+            failed query information.
+
+    Returns:
+        List of dictionary of
```

Comment:
```suggestion
        List of dictionarified 
```

Dictonarified instead of dictionary of. 

For all other APIs too. (You can use a word better than "dictionarified" if anything comes up to you lol..)